frontend/ui/screensaver.lua

```diff
@@ -147,4 +147,25 @@ function Screensaver:close()
     end
 end
 
+function Screensaver:isKindleSpecialOffers()
```

Comment:
This is a device specific function, so it should live in the device module instead. Screensaver is a UI component that should be decoupled from device model logic.